managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerFactoryConfig.java

```diff
@@ -92,8 +92,30 @@ public class ManagedLedgerFactoryConfig {
      */
     private String managedLedgerInfoCompressionType = MLDataFormats.CompressionType.NONE.name();
 
+    /**
+     * ManagedLedgerInfo compression threshold. If the origin metadata size below configuration.
+     * compression will not apply.
+     */
+    private int managedLedgerInfoCompressionSizeThresholdBytes = 0;
```

Comment:
Should we set a suitable default value?

Reply:
current value is the same behavior before this feature. maybe we can set to 128 or more. In our production env most cursor metadata is less than this value.